Packs/CofenseTriage/Integrations/CofenseTriagev3/README.md

```diff
@@ -0,0 +1,2436 @@
+- `operator` is the comparison operator to use when comparing the attribute to the specified value. The default is EQ. You can omit
+this parameter if you want to use the default. See Comparison Operators for the expected syntax for other comparison
+operators.
+- `value` is the value being checked for. You can specify multiple values as a comma-separated list. Doing so returns records
+that match _ANY_ of the supplied values.
+- To specify multiple filters, use the comma ( , ) to separate them 
+  (for example, `{\"attribute1_operator \": \"value1, value2\", \"attribute2_operator\" : \"value3, value4\"}`). 
+  Doing so returns only those records that match _ALL_ the filters.
+- A filter can be applied against the same attribute (and operator) multiple times.
+
+## Comparison Operators
+
+### Standard Comparison Operators
+- `eq`: This is the default comparison operator if no operator is specified. Returns results when an attribute is equal to the supplied value.
+- `not_eq`: Returns results when an attribute is not equal to the supplied value.
+- `lt`: Returns results when an attribute is less than the supplied value.
+- `lteq`: Returns results when an attribute is less than or equal to the supplied value.
+- `gt`: Returns results when an attribute is greater than the supplied value.
+- `gteq`: Returns results when an attribute is greater than or equal to the supplied value.
+
+### String Comparison Operators
+In addition to the standard comparison operators, string attributes can also use the following comparison operators:
+- `start`: Returns results when an attribute starts with the supplied value.
+- `not_start`: Returns results when an attribute does not start with the supplied value.
+- `end`: Returns results when an attribute ends with the supplied value.
+- `not_end`: Returns results when an attribute does not end with the supplied value.
+- `cont`: Returns results when an attribute contains the supplied value.
+- `not_cont`: Returns results when an attribute does not contain the supplied value.
+
+### Array Comparison Operators
+Some resources have an array attribute that contains a list of values. Array attributes follow a common usage pattern, but the
+filters supported will vary.
+- `any_OP`: Returns resources where any value in the array matches the standard or string comparison operator (`OP`).
+
+### Tag List Comparison Operators
+Some resources have a `tag_list` attribute that contains a list of Triage tags that were applied to that resource.
+Tag List attributes do not support the standard or string comparison operators. Attributes of this type support the following
+comparison operators:
+- `any`: Returns results when a resource is tagged with any of the specified tags.
+- `all`: Returns results when a resource is tagged with all of the specified tags.
+- `none`: Returns results when a resource is not tagged with any of the specified tags.
+
+## Mirroring Integration Feature
```

Comment:
also please note here that it is required to see the mirroring direction parameter